sdks/workflows.do/src/index.ts

```diff
@@ -27,6 +28,47 @@ import type {
 export function AI<T extends AIConfig>(config: T): AIInstance {
   const instance: Record<string, AIFunction> = {}
   
+  const registrationPromises = Object.entries(config).map(async ([key, value]) => {
+    if (typeof value === 'function') {
+      const functionString = value.toString()
+      
+      const response = await fetch('https://apis.do/workflows/register', {
```

Comment:
This path should be dynamic, because when running/testing locally it will be http://localhost:3000 and when live it will be https://apis.do/ ... while the staging environment is https://apis.do.sg/